src/ComponentCatalogPanel.js

```diff
@@ -2,17 +2,60 @@ import React, {Component} from 'react';
 import {imagePath} from './constants';
 import {MDCRipple} from '@material/ripple/index';
 import HeroComponent from './hero/HeroComponent';
+import {HeroOptionsComponent} from './hero/HeroOptionsComponent';
+import equal from 'deep-equal';
+import {getUrlParamsFromSearch} from './hero/urlHelper';
+
+import './styles/ComponentCatalogPanel.scss';
 
 // ComponentCatalogPanel is the container for catalog component content,
 // that renders the hero and demo sections.
 class ComponentCatalogPanel extends Component {
   ripples = [];
   initRipple = el => el && this.ripples.push(new MDCRipple(el));
 
+  constructor(props) {
+    super(props);
+    // Deep copy for local object
+    const localConfig = JSON.parse(JSON.stringify(this.props.initialConfig));
```

Comment:
This causes issues with component Hero sections that don't have options.

Reply:
ah...ok I'll fix that. Do the rest of the changes look ok to you?

Reply:
Yeah, I think it looks great. I pushed some styles for max width so the card web tab doesn't expand to the entire screen but I definitely think it looks better.